scripts/feature_tests/cleanup_env.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 set -x
 
+ROOTPATH="$(dirname "$(readlink -f "${0}")")/../.."
+
+# shellcheck disable=SC1091
+source "${ROOTPATH}/scripts/feature_tests/feature_test_vars.sh"
```

Comment:
Let's make it consistent with the lines sourcing lib/* scripts below. Either prefix them with `${ROOTPATH}/`, or skip it here and still rely on the script being run from metal3-dev-env root.

Reply:
Since it is cosmetic change not a functionality change and it takes ~3h to pass feature tests, I would suggest merge this as is to unblock CI and improve it future patch, WDYT?

Reply:
Sure, go ahead

Reply:
> improve it future patch

@wgslr see #718